frontend/src/modules/Survey/Components/Survey.tsx

```diff
@@ -6,51 +6,69 @@ import { StepBegin } from 'Survey/Components/StepBegin'
 import { StepCourse } from 'Survey/Components/StepCourse'
 import { StepRadio } from 'Survey/Components/StepRadio'
 import { StepFinal } from 'Survey/Components/StepFinal'
-import {
-  sendSurveyData,
-  SurveyData,
-} from 'Survey/Components/FuncsAndConsts/SurveyFunctions'
+import { SurveyData } from 'Survey/Components/FuncsAndConsts/SurveyFunctions'
 import { Question } from '@core/Types'
+import axios from 'axios'
+import { API_ROOT, STUDENT_API } from '@core/Constants'
 
 export const Survey = () => {
   const [showError, setShowError] = useState(false)
-  const [currStep, setCurrStep] = useState(0)
+  const [currStep, setCurrStep] = useState(1)
+  const [surveySuccess, setSurveySuccess] = useState(false)
+  const [surveyError, setSurveyError] = useState('')
```

Comment:
I think surveySuccess is redundant because if there's an error message that means it was not successful. You could use `useState<string | null>(null)` and check if the error is null or not